<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bitsetLabel1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 18pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>bitset 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="bitsetLabel2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 18pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>bitset 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="bitsetLabel3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 18pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>bitset 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="set1lineEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="set2lineEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>set</string>
    </property>
   </widget>
   <widget class="QLabel" name="someLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 18pt &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Some Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>first bitset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>second bitset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Papyrus&quot;;</string>
    </property>
    <property name="text">
     <string>result bitset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
